charts/ingress-nginx/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: ingress-nginx
-version: 3.7.1
+version: 3.8.0
```

Comment:
Please do not update the version.
(I want to group several PRs to the chart)

Reply:
updated